docs/Custom_Device_Types.md

```diff
@@ -0,0 +1,184 @@
+## Custom Device Types
+
+While deployed to the Target, all custom devices run inside the VeriStand Engine. The engine is the non-visible mechanism that controls the timing of the entire system as well as communication between the Target and Host Computer. See **[VeriStand Manual](https://www.ni.com/documentation/en/veristand/latest/manual/manual-overview/)** » **[VeriStand Environment](https://www.ni.com/documentation/en/veristand/latest/manual/environment/)** » **[Components of a VeriStand Project](https://www.ni.com/documentation/en/veristand/latest/manual/project-components/)** » **[VeriStand Engine](https://www.ni.com/documentation/en/veristand/latest/manual/vs-engine/)** for more information.
+
+The [niveristand-custom-device-wizard](https://github.com/ni/niveristand-custom-device-wizard/releases) generates a new LabVIEW Project containing one of five [device frameworks](https://www.ni.com/documentation/en/veristand/latest/manual/custom-device-types/). The framework is determined by the **Custom Device Execution Mode** control.
+
+![](images/Execution_mode_control.jpg)
+
+**Figure: NI VeriStand Custom Device Wizard**
```

Comment:
Generally, when showing an image it is labeled with Figure 1, Figure 2, Figure 3.... and so on down the page. That way the body text can refer to a specific image.  Can you add numbers to these?